### PR TITLE
test: move functions from utils

### PR DESCRIPTION
**Pull request by codingben**
Move functions from utils to make it shorter.

Signed-off-by: boukhanov@redhat.com

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Review by maya-r — APPROVED** [review]

File: tests/BUILD.bazel [hunk 1]
```diff
@@ -19,7 +19,6 @@ go_library(
         "//pkg/util/net/ip:go_default_library",
         "//pkg/virt-config:go_default_library",
         "//pkg/virt-controller/services:go_default_library",
-        "//pkg/virt-handler/cgroup:go_default_library",
```
[the remark this answers is not in the document]
Comment (codingben) [reply tests/BUILD.bazel:22]: Alright (you won't squash commits?).

**Comment by codingben** [comment]
@xpivarc As for other comments, I agree, but it should be done in another PR.

File: tests/BUILD.bazel (hunk 1 above)
[the remark this answers is not in the document]
Comment (xpivarc) [reply tests/BUILD.bazel:22]: Sorry, I don't follow. Can you elaborate?

**Comment by xpivarc** [comment]
> @xpivarc As for other comments, I agree, but it should be done in another PR.

I personally think that inlining is an alternative option to `move` and not follow up. But I will not insist if you will follow up.

File: tests/BUILD.bazel (hunk 1 above)
[the remark this answers is not in the document]
Comment (codingben) [reply tests/BUILD.bazel:22]: We'll combine multiple commits here into one commit, no?

[the remark this answers is not in the document]
Comment (codingben) [reply tests/BUILD.bazel:22]: I'll resolve this comment tomorrow.

**Comment by xpivarc** [comment]
/approve
/hold
Feel free to unhold when you resolve the comment.

File: tests/BUILD.bazel (hunk 1 above)
[the remark this answers is not in the document]
Comment (codingben) [reply tests/BUILD.bazel:22]: That commit was removed. I went through each commit and ran `make bazel-generate`, if I saw changes to the file I edited that commit. Please re-review.

**Comment by codingben** [comment]
@xpivarc Can you please re-review? comment resolved.

File: tests/vmi_ignition_test.go [hunk 2]
```diff
@@ -90,3 +91,7 @@ var _ = Describe("[rfe_id:151][crit:high][vendor:cnv-qe@redhat.com][level:compon
 
 	})
 })
+
+func hasExperimentalIgnitionSupport() bool {
```
[the remark this answers is not in the document]
Comment (codingben) [reply tests/vmi_ignition_test.go:95]: Resolved in #7903.

File: tests/security_features_test.go [hunk 3]
```diff
@@ -248,13 +255,33 @@ var _ = Describe("[Serial][sig-compute]SecurityFeatures", func() {
 
 			By("Checking virt-launcher Pod's compute container has precisely the documented extra capabilities")
 			for _, capa := range caps.Add {
-				Expect(tests.IsLauncherCapabilityValid(capa)).To(BeTrue(), "Expected compute container of virt_launcher to be granted only specific capabilities")
```
[the remark this answers is not in the document]
Comment (codingben) [reply tests/security_features_test.go:251]: Hmm, yes. So just to confirm, you was thinking about dropping that test marked [here](https://github.com/kubevirt/kubevirt/blob/main/tests/security_features_test.go#L256-L259)?

**Pull request merged**